blog/releases/announcing-knative-v0-18-release.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Version v0.18 release"
+title: "Version 0.18 release"
```

Comment:
see above and https://docs.google.com/spreadsheets/d/1p1_kBUd6ZvonxHkMcEJPayf6QIpExuFf5cFq0ptar7I/edit#gid=0